README.md

```diff
@@ -74,18 +88,18 @@ Wasa2il must be set up on a web server capable of running Django. Instructions o
 
 If you have `docker-compose` installed, you need to:
 
-1. Start web + db containers:  
+1. Start web + db containers:
```

Comment:
Right.. my editor strips trailing whitespace for better or worse...
Well - for better IMO, since doing this tends to make git patches easier to traverse in the long-run :)
Feel free to push against it, and I'll configure my vim not to do that in this project.

Reply:
Is it better to do a single commit that addresses all trailing whitespace issues?
Instead of allowing one and one whitespace issue in multiple commits

Reply:
Hah, I was thinking of that.. The whitespace thing was a bit of an afterthought in this PR - but I could totally do a global whitespace trim across the repo, and put that in it's own PR for sure! I'll look into it tonight (Pacific time).

Reply:
Yep! Re-added the whitespace, and I found a nice way to strip whitespace globally. I'll aim to do that in a follow-up PR

Based on: https://stackoverflow.com/questions/149057/how-to-remove-trailing-whitespace-of-all-files-recursively

`git grep -I --name-only -z -e '' | xargs -0 sed -i 's/[ \t]\+\(\r\?\)$/\1/'`

Seems to do the trick nicely!